source/quickguides/quick-guide-chrysalis.rst

```diff
@@ -0,0 +1,268 @@
+Chrysalis quick guide for running e3sm_diags v2
```

Comment:
We should probably include information parallel to Cori and Compy for `source/quickguides/quick-guide-multiprocessing-chrysalis.rst`. I imagine people do intend to use multiprocessing on Chrysalis. That file isn't auto-generated by the Python script.

Reply:
Got it, I will take a closer look at this

Reply:
Hey Tom, here is info on Chrysalis https://acme-climate.atlassian.net/wiki/spaces/ED/pages/1501102232/Chrysalis+-+E3SM+s+1PF+AMD+system, it uses `srun` for requesting interactive session on compute nodes.